### ISSUE TITLE
Links are not clearly identifiable

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9042]
# Motivation

Getting a SI level A error on Academic Calendar and Five Year Calendar

- Links are not clearly identifiable
  - 1.4.1 Use of Color
    - Color alone is not enough to distinguish a link from the surrounding text.
    - Add a secondary visual identifier for links.

# Additional Info

https://my2.siteimprove.com/Inspector/995867/A11Y/Page?pageId=25148795285&impmd=264ED37B65FA6186129334A508391970&issue=cantTell+failed&wcag=twopointone#/sia-r62/failed

This link may not work anymore ^^.

Go to https://registrar.uiowa.edu/academic-calendar, search commencement.uiowa.edu and use SI browser plugin to view errors.

Go to https://registrar.uiowa.edu/five-year-calendar#2025-26, use SI browser plugin to view errors.

**Comment by joewhitsitt** [comment]
Not seeing this issue on this link anymore.

**Comment by briand44** [comment]
The link in the issue is giving me problems today. Going to add additional info to IS.

**Comment by bspeare** [comment]
The fix for this is at https://github.com/uiowa/uiowa/blob/main/docroot/sites/registrar.uiowa.edu/modules/registrar_core/src/Controller/AcademicCalendarController.php#L439-L452.  The description should render in the body and not the meta field.  I think we only put it in the meta field originally because the spacing looked a little better. The session badges and the description should both render in the body instead.

**Comment by e-marie-w** [comment]
Adding the event_description to the `#content` section works great for getting the underline to apply. But the spacing between the description and the badges is now reduced (since they're now in the same section) so I'm trying to fix that. 

<img width="1108" height="557" alt="Image" src="https://github.com/user-attachments/assets/683c81c1-fabb-489d-ab48-97c3f169c0d2" />
